projects/microsoft.com/code-cli/package.yml

```diff
@@ -0,0 +1,31 @@
+distributable:
+  url: https://github.com/microsoft/vscode/archive/refs/tags/{{version.tag}}.tar.gz
+  strip-components: 1
+
+displaty-name: vscode cli
+
+versions:
+  github: microsoft/vscode
+
+dependencies:
+  openssl.org: 1.1
+  zlib.net: 1
+
+build:
+  dependencies:
+    rust-lang.org: ^1.81
+    rust-lang.org/cargo: '*'
+  working-directory: cli
+  script: cargo install --locked --path . --root {{prefix}}
+  env:
+    OPENSSL_NO_VENDOR: 1
+    OPENSSL_DIR: '{{deps.openssl.org.prefix}}'
+    VSCODE_CLI_NAME_LONG: Code OSS
```

Comment:
Just wondering... Why you changed from the default _Code - OSS_?

https://github.com/microsoft/vscode/blob/f14abad992a026eef3c83491c59b3e2628b99f66/cli/src/constants.rs#L58

Reply:
Generally observe homebrew's methodology. You're correct that it shouldn't be needed.